### DISCUSSION TITLE
Farming the same plot in 2 devices

### DISCUSSION THREAD

**Discussion by kelvin199602** [Chia-Network/chia-blockchain#2335] [Q&A]
I was moving the plot from one device to another, but what would happen if I farm the same plot in both devices? (with the same key) is it going to cause any bugs or it would make it farm faster?

**Comment by leoliyuanduo** [comment]
That's an interesting idea. I doubt it would make it farm faster. I assume only one plot would sync. I do not have another computer to test it. Hopefully, someone can answer it.

**Reply by kelvin199602** [reply]
interestingly it would sync without any problems but just i have no idea is it really farming. (the speed of solving block is still low and i have no luck)

**Reply by leoliyuanduo** [reply]
Good to know. https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Architecture

**Reply by Rafgt** [reply]
Hi, i am thinking about plotting drives on 3 devices and then split them in one device. You have been moving the plot from one device to another. Have you been using GUI? Did you use the same private key on both devices?

**Reply by kelvin199602** [reply]
Have you been using GUI? -Yes
Did you use the same private key on both devices? -Yes
It works fine

**Reply by Rafgt** [reply]
Thx:) I will do the same.
